app/models/tag.rb

```diff
@@ -87,7 +87,7 @@ def calculate_hotness_score
     self.hotness_score = Article.tagged_with(name).
       where("articles.featured_number > ?", 7.days.ago.to_i).
       map do |article|
-        (article.comments_count * 14) + (article.reactions_count * 4) + rand(6) + ((taggings_count + 1) / 2)
+        (article.comments_count * 14) + article.positive_reactions_count + rand(6) + ((taggings_count + 1) / 2)
```

Comment:
```suggestion
        (article.comments_count * 14) + article.score + rand(6) + ((taggings_count + 1) / 2)
```

I'm good to go with as is, but this is all I was proposing. `score` is basically the sum of positive and negative reactions on  a post.